plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraQueryRunner.java

```diff
@@ -56,6 +56,8 @@ public static DistributedQueryRunner createCassandraQueryRunner(
         queryRunner.createCatalog("cassandra", "cassandra", ImmutableMap.of(
                 "cassandra.contact-points", server.getHost(),
                 "cassandra.native-protocol-port", Integer.toString(server.getPort()),
+                "cassandra.load-policy.use-dc-aware", "true",
+                "cassandra.load-policy.dc-aware.local-dc", "datacenter1",
```

Comment:
Could you leave this behavior change to documentation and commit body?

Reply:
I don't exactly understand what you mean.

Reply:
These config properties were not required before this change. Please document it.

Reply:
I've modified the commit comment.

When not setting these two properties, we receive now the following exception:

```
Caused by: java.lang.IllegalStateException: Since you provided explicit contact points, the local DC must be explicitly set (see basic.load-balancing-policy.local-datacenter in the config, or set it programmatically with SessionBuilder.withLocalDatacenter). Current contact points are: Node(endPoint=localhost/127.0.0.1:63985, hostId=56de3092-7ca1-4b63-a9b1-965415cc07fd, hashCode=797c3de3)=datacenter1. Current DCs in this cluster are: datacenter1
```

As you can see the exception is rather misleading because `basic.load-balancing-policy.local-datacenter` is a configuration specific to the Datastax Cassandra driver. See https://docs.datastax.com/en/developer/java-driver/4.11/manual/core/load_balancing/

I'm somehow reluctant that these properties will help the operators of this connector to setup the connector correctly.

Any thoughts on how to deal with such exceptions?